passes/cmds/plugin.cc

```diff
@@ -36,14 +44,48 @@ void load_plugin(std::string filename, std::vector<std::string> aliases)
 	if (filename.find('/') == std::string::npos)
 		filename = "./" + filename;
 
+	#ifdef WITH_PYTHON
+	if (!loaded_plugins.count(filename) && !loaded_python_plugins.count(filename)) {
+	#else
 	if (!loaded_plugins.count(filename)) {
+	#endif
+
+		#ifdef WITH_PYTHON
+		if(boost::algorithm::ends_with(filename, ".py"))
+		{
+			int last_slash = filename.find_last_of('/');
```

Comment:
Should this consider Windows where '\' might be used? I can see that the other plugin related code doesn't consider this, but it also seems to rely on dlopen so won't work on Windows in any case - whereas I guess the Python plugin interface should be cross-platform.

Reply:
Solved using the experimental c++ std library filesystem. fd7fb1377d4d30d692c78eb55173198339fea17d